ext/native/ext/vjson/main.cpp

```diff
@@ -25,7 +25,7 @@ void populate_sources(const char *filter, std::vector<std::vector<char> > &sourc
 		}
 	}
 
-	printf("Loaded %d json files\n", sources.size());
+	printf("Loaded %d json files\n", (int)sources.size());
```

Comment:
These changes look like bugfixes, albeit to code we don't actually use.  It looks like upstream was already changed differently:

https://github.com/vivkin/vjson/blob/master/main.cpp#L32

-[Unknown]